ge/org.osate.ge.ba/src/org/osate/ge/ba/ui/properties/BehaviorTransitionPropertySection.java

```diff
@@ -0,0 +1,231 @@
+package org.osate.ge.ba.ui.properties;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.BiFunction;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.Adapters;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.edit.domain.AdapterFactoryEditingDomain;
+import org.eclipse.emf.transaction.TransactionalEditingDomain;
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.jface.layout.GridLayoutFactory;
+import org.eclipse.jface.viewers.IFilter;
+import org.eclipse.jface.viewers.ISelection;
+import org.eclipse.jface.window.Window;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.custom.StyledText;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.ui.IWorkbenchPart;
+import org.eclipse.ui.views.properties.tabbed.AbstractPropertySection;
+import org.eclipse.ui.views.properties.tabbed.TabbedPropertySheetPage;
+import org.eclipse.xtext.resource.XtextResource;
+import org.eclipse.xtext.ui.editor.model.IXtextDocument;
+import org.eclipse.xtext.xbase.lib.Pair;
+import org.osate.ba.aadlba.BehaviorCondition;
+import org.osate.ba.aadlba.BehaviorTransition;
+import org.osate.ba.declarative.DeclarativeBehaviorTransition;
+import org.osate.ge.BusinessObjectContext;
+import org.osate.ge.BusinessObjectSelection;
+import org.osate.ge.ProjectUtil;
+import org.osate.ge.ba.util.BehaviorAnnexSelectionUtil;
+import org.osate.ge.internal.services.ActionExecutor.ExecutionMode;
+import org.osate.ge.internal.services.ActionService;
+import org.osate.ge.internal.services.ModelChangeNotifier;
+import org.osate.ge.internal.ui.xtext.AgeXtextUtil;
+import org.osate.ge.swt.SwtUtil;
+import org.osate.ge.ui.PropertySectionUtil;
+
+/**
+ * Property section for {@link BehaviorTransition}
+ */
+public class BehaviorTransitionPropertySection extends AbstractPropertySection {
```

Comment:
If I select multiple transitions. The text for the condition of one of them shows in the property section. The edit button is disabled. At the least, the field should have text that indicates that the value for all selections is not the same when the values for the selected transitions are not equal. The property section could also be hidden instead.